index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
+    <title>Adel Guitoun - Portfolio</title>
+  </head>
+  <body class="bg-decoration">
+    <header class="section header-section">
+      <div class="container">
+        <button class="menu">
+          <img src="./assets/icons/menu.svg" alt="Menu" />
+        </button>
```

Comment:
- [ ] Kindly wrap your hamburger icon into the `nav` tag which will not only maintain the correct usage of semantics, it will even increase the user experience for mobile devices. :wink:

Reply:
Thank you for this, totally agree.